kak-lsp.toml

```diff
@@ -83,6 +83,11 @@ icons = "unicode"
 isHttpEnabled = true
 statusBarProvider = "log-message"
 compilerOptions = { overrideDefFormat = "unicode" }
+inlayHints.hintsInPatternMatch.enable = true
+inlayHints.implicitArguments.enable = true
+inlayHints.implicitConversions.enable = true
+inlayHints.inferredTypes.enable = true
+inlayHints.typeParameters.enable = true
```

Comment:
Thanks! I wonder why these are not enabled by default

Reply:
Yeah, I don't know, at least the last 2, for example as they are quite minimal and not cluttering.